partner_priority/README.rst

```diff
@@ -0,0 +1,25 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: LGPL-3
+
+===========================
```

Comment:
Missed this one on first review. The `===` lines above and below a title, should be exactly as long as the title.

Reply:
Done